platforms/common/uORB/uORBDeviceMaster.cpp

```diff
@@ -150,7 +150,7 @@ int uORB::DeviceMaster::advertise(const struct orb_metadata *meta, bool is_adver
 
 			// add to the node map.
 			_node_list.add(node);
-			_node_exists[node->get_instance()].set((uint8_t)node->id(), true);
+			_node_exists[node->get_instance()].set((orb_id_size_t)node->id(), true);
```

Comment:
Are nodes the same as topics? Or do nodes send and receive any number of topics? In which case this doesn't need to be incremented? thanks